dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -81,6 +81,9 @@ def profile(self):
             profile["statistics"].update(
                 dict(categories=self.categories)
             )
+            profile["statistics"].update(
+                dict(gini_impurity=self.gini_impurity)
+            )
```

Comment:
I believe you could just add the gini_impurity key to the call to update above, with the categories.